lib/src/render.dart

```diff
@@ -0,0 +1,7 @@
+// Copyright 2017 Google Inc. Use of this source code is governed by an
+// MIT-style license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+export 'render/interface.dart'
+    if (dart.library.io) 'render/vm.dart'
+    if (node) 'render/node.dart';
```

Comment:
This isn't necessary. The API exposed by `lib/sass.dart` is only relevant to Dart. The API that's seen by JS is produced by the exports assigned in `lib/src/node.dart`. That may change in the future as JS interop gets more fully-featured, but for now it doesn't make sense to export any Node APIs through `lib/sass.dart`.